### PR TITLE
Job to release jars to Maven without community release.

### PR DESCRIPTION
**Pull request by alenkacz**
I think the way Metronome is integrated with Marathon won't change in upcoming weeks/months. This is a job that automates one part of the integration. We usually want to bump earlier than we do community release.

JIRA issues: DCOS-50471

### CODE REVIEW

**Comment by alenkacz** [comment]
example run https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/marathon-team-releases/job/release-jars/2/console

**Comment by jeschkies** [comment]
`release jars` is a little misleading IMHO since Marathon itself is released to DC/OS as a Jar as well. I have to say that I was never a fan of this integration. Marathon should just not be released as a library to Maven. Or am I missing something and this is just another S3 upload?

File: Jenkinsfile.releasejars [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY'),
+        ]) {
+          sshagent (credentials: ['0f7ec9c9-99b2-4797-9ed5-625572d5931d']) {
+            sh """sudo -E env sbt 'project plugin-interface' publish"""
+            sh """sudo -E env sbt publish"""
+          }
+        }
+      } finally {
+        junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
```
Comment (zen-dog) [inline Jenkinsfile.releasejars]: I think this whole section does not apply to `sbt publish` issued above.

**Comment by alenkacz** [comment]
@jeschkies I renamed it to publish jars, hope that helps.

Yes, this is far from ideal but this is that kind of integration we settled on between marathon and metronome. We had discussion about merging the repos and we never pursued that but we absolutely want to keep those two up to date to not create more tech debt. This job is just allowing us to do that. We are doing this these days anyway because of metronome so why hide it and always have to figure out how to integrate those two.

**Comment by jeschkies** [comment]
Thanks for the renaming.

**Pull request merged**